wazimap/geo.py

```diff
@@ -285,9 +285,10 @@ def get_locations_from_coords(self, longitude, latitude, levels=None, version=No
 
         for features in self.geometry.values():
             for feature in features.values():
-                if feature['shape'] and feature['shape'].contains(p):
-                    geo = self.get_geography(feature['properties']['code'],
-                                             feature['properties']['level'],
+                feature_key = list(feature.keys())[0]
```

Comment:
I was unsure of a variable name to use. Please suggest if any changes are to be made.

Reply:
That looks fine.